layersvt/device_simulation.md

```diff
@@ -95,32 +95,71 @@ JSON file formats consumed by the DevSim layer are specified by one of the JSON
 |:----------:|:-------------:|
 | Vulkan v1.0 | https://schema.khronos.org/vulkan/devsim_1_0_0.json# |
 | Vulkan v1.1 | https://schema.khronos.org/vulkan/devsim_1_1_0.json# |
+| Vulkan v1.2 | https://schema.khronos.org/vulkan/devsim_1_2_0.json# |
```

Comment:
Is there a mechanism to express that an entire extension is not supported?

Reply:
I'm not sure what you had in mind.
I'd think the fact that none of an extensions queries are defined in the schema would be enough of a flag to a user that said extension queries are not supported.

Reply:
A devsim tool should be to amend the list of reported extensions.  Considering that devsim is a "device simulator", the idea would be to able to check whether my Vulkan applications can run on a less capable device.